lib/iris/tests/idiff.py

```diff
@@ -55,6 +56,7 @@
 _POSTFIX_JSON = os.path.join('results', 'imagerepo.json')
 _POSTFIX_LOCK = os.path.join('results', 'imagerepo.lock')
 _PREFIX_URI = 'https://scitools.github.io/test-images-scitools/image_files'
```

Comment:
this is only used once, could we have it used in line, not obfuscated in a global please?